docs/release-notes/self-hosted/self-hosted-v11.0.0.md

```diff
@@ -0,0 +1,94 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v11.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.3
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.6
+---
+
+# Self-hosted v11.0.0
+
+These release notes are for [Codacy Self-hosted v11.0.0](https://github.com/codacy/chart/releases/tag/11.0.0){: target="_blank"}, released on April 19, 2023.<!-- TODO Update release date -->
```

Comment:
Should this date also be updated?

Reply:
Yes :+1:, I usually do that as the very last thing before merging the pull request because it's at that time that we're 100% sure of the release date. :sweat_smile: